plugins/org.jboss.tools.openshift.core/META-INF/MANIFEST.MF

```diff
@@ -1,7 +1,7 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
-Bundle-SymbolicName: org.jboss.tools.openshift.core
+Bundle-SymbolicName: org.jboss.tools.openshift.core;singleton:=true
```

Comment:
why force this plugin to be a singleton? only plugins with extension points or UI need this?